setup.sh

```diff
@@ -70,6 +70,7 @@ pssh --inline \
     --host "$MASTERS $SLAVES" \
     --user root \
     --extra-args "-t -t $SSH_OPTS" \
+    --timeout 300 \
```

Comment:
@tomerk What timeout do you see without this setting? The man page doesn't mention a default value.

It does mention, though, that you can set the timeout to 0 and pssh will wait until the called processes complete. Does that not work here?